### ISSUE TITLE
Combine `conda env create` with `conda create` and make the former an alias

### ISSUE THREAD

**Issue by jezdez** [conda/conda#13015]
We desire consistent behavior between `conda create` and `conda env create`.

Some of the inconsistencies between the two commands:

| feature | `conda create` | `conda env create` |
|---|---|---|
| list dependencies on CLI | ✅ | ❌ |
| `requirements.txt` | ✅ | ✅ |
| `environment.yaml` | ❌ | ✅ |
| remote `environment.yaml` | ❌ | ✅ |
| specify additional channels (`--channel=CHANNEL`) | ✅ | ❌ |
| override existing channels (`--override-channels`) | ✅ | ❌ |

**Comment by beeankha** [comment]
Connecting https://github.com/conda/conda/pull/14113 here

**Comment by jaimergp** [comment]
xref https://github.com/conda/conda/pull/14446

**Comment by danyeaw** [comment]
Not stale.

**Comment by janxkoci** [comment]
I need to create an environment using a yml file, but conda does not let me - is there any workaround? Version 25.11.0, installed last week.

PS: sorry for the multiple messages, GitHub moving to Azure has some glitches, I had to use phone because GitHub didn't have free server to log me in on a laptop - ridiculous!

PPS: I see errors as in https://github.com/conda/conda/issues/13676. The workaround is to use mamba.

**Comment by GF-Huang** [comment]
Issue still 2026.

**Comment by janxkoci** [comment]
Yes, the functionality is currently still broken. However, my understanding is that only YML support is broken, while other [specification formats](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#sharing-an-environment), like txt or json, may work.

Or as I said above - you can use mamba with YML files.
